src/main/java/com/uber/cadence/internal/common/WorkflowExecutionUtils.java

```diff
@@ -621,26 +437,6 @@ public static GetWorkflowExecutionHistoryResponse getHistoryPage(
     return history;
   }
 
-  /** Returns workflow instance history in a human readable format. */
-  public static String prettyPrintHistory(
```

Comment:
This could be useful; I wonder why have you decided to drop this.

Reply:
I think it's potentially useful but it's long unused and untested. If we need similar functionality we can rewrite it.